boards/DIM/Inc/Io/Io_SevenSegDisplays.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <stdbool.h>
+#include "App_SevenSegDisplay.h"
+
+void Io_SevenSegDisplays_Init(SPI_HandleTypeDef *hspi);
+void Io_SevenSegDisplays_WriteCommands(void);
```

Comment:
@garethellis0 Got any idea where we should be calling this function? I gave it some thought but wasn't too sure.